src/main/java/at/hannibal2/skyhanni/features/summonings/SummoningMobManager.kt

```diff
@@ -2,55 +2,50 @@ package at.hannibal2.skyhanni.features.summonings
+import at.hannibal2.skyhanni.events.MobEvent
 import at.hannibal2.skyhanni.events.SkyHanniRenderEntityEvent
-import at.hannibal2.skyhanni.mixins.hooks.RenderLivingEntityHelper
 import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
 import at.hannibal2.skyhanni.utils.ChatUtils
-import at.hannibal2.skyhanni.utils.ColorUtils.withAlpha
-import at.hannibal2.skyhanni.utils.EntityUtils
-import at.hannibal2.skyhanni.utils.LocationUtils
+import at.hannibal2.skyhanni.utils.DelayedRun
 import at.hannibal2.skyhanni.utils.LorenzColor
 import at.hannibal2.skyhanni.utils.LorenzUtils
 import at.hannibal2.skyhanni.utils.LorenzUtils.baseMaxHealth
-import at.hannibal2.skyhanni.utils.NumberUtil
+import at.hannibal2.skyhanni.utils.MobUtils.mob
 import at.hannibal2.skyhanni.utils.NumberUtil.shortFormat
-import at.hannibal2.skyhanni.utils.RegexUtils.matchMatcher
-import at.hannibal2.skyhanni.utils.RenderUtils.renderStrings
-import at.hannibal2.skyhanni.utils.getLorenzVec
+import at.hannibal2.skyhanni.utils.RegexUtils.matches
+import at.hannibal2.skyhanni.utils.RenderUtils.renderRenderable
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import at.hannibal2.skyhanni.utils.renderables.Renderable
 import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
-import net.minecraft.entity.EntityLiving
-import net.minecraft.entity.EntityLivingBase
 import net.minecraft.entity.item.EntityArmorStand
 import net.minecraftforge.fml.common.eventhandler.EventPriority
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.seconds
 
 @SkyHanniModule
 object SummoningMobManager {
 
     private val config get() = SkyHanniMod.feature.combat.summonings
+    private var mobs = mutableSetOf<Mob>()
 
-    private val summoningMobs = mutableMapOf<EntityLiving, SummoningMob>()
-    private val summoningMobNametags = mutableListOf<EntityArmorStand>()
-    private var summoningsSpawned = 0
-    private var searchArmorStands = false
-    private var searchMobs = false
+    private var lastChatTime: SimpleTimeMark = SimpleTimeMark.farPast()
+    private val timeOut = 2.seconds
 
     private val patternGroup = RepoPattern.group("summoning.mobs")
-    private val spawnPattern by patternGroup.pattern( //§aYou have spawned your Tank Zombie §r§asoul! §r§d(249 Mana)
+
+    /**
+     * REGEX-TEST: §aYou have spawned your Tank Zombie §r§asoul! §r§d(249 Mana)
+     */
+    private val spawnPattern by patternGroup.pattern(
         "spawn",
-        "§aYou have spawned your (.+) §r§asoul! §r§d\\((\\d+) Mana\\)"
+        "§aYou have spawned your (.+) §r§asoul! §r§d\\((\\d+) Mana\\)",
     )
     private val despawnPattern by patternGroup.pattern(
         "despawn",
-        "§cYou have despawned your (monster|monsters)!"
-    )
-    private val healthPattern by patternGroup.pattern( //§a§ohannibal2's Tank Zombie§r §a160k§c❤
-        "health",
-        "§a§o(.+)'s (.+)§r §[ae]([\\dkm]+)§c❤"
+        "§cYou have despawned your monsters?!",
```

Comment:
`?` feels wrong here. please add a regex101 test to make it clear the original message is not the literal test string
![grafik](https://github.com/hannibal002/SkyHanni/assets/24389977/b69f81e7-5f78-4dab-927a-87001ff628e6)